eng/illink.targets

```diff
@@ -26,9 +26,6 @@
   <!-- Inputs and outputs of ILLinkTrimAssembly -->
   <PropertyGroup>
     <ILLinkDirectory Condition="'$(ILLinkDirectory)' == ''">$(MSBuildProjectDirectory)\ILLink\</ILLinkDirectory>
-    <ILLinkTasksDir>$([MSBuild]::NormalizeDirectory('$(PkgMicrosoft_NET_ILLink_Tasks)', 'tools'))</ILLinkTasksDir>
```

Comment:
Should you also remove the GeneratePathProperty from the above packageReference?

Reply:
Oops, my mistake. The above PackageReference needs to be removed. Just fixed that.